src/main.h

```diff
@@ -33,32 +33,20 @@ class Nmea2000 : public tNMEA2000 {
   bool CANOpen() override;
   bool CANGetFrame(unsigned long &id,
                            unsigned char &len, unsigned char *buf) override;
-
  private:
-  NodeNmea2000* nodeNmea2000_;
-  std::deque<Packet> packets_;
-};
 
-class NodeNmea2000 : public Nan::ObjectWrap {
- public:
-  NodeNmea2000(): nmea2000_(this) { }
-
-  static void Init(v8::Local<v8::Object> exports);
-
-  static NAN_METHOD(New);
-  static NAN_METHOD(pushCanFrame);
-  static NAN_METHOD(setSendCanFrame);
-  static NAN_METHOD(open);
-  static NAN_METHOD(parseMessages);
+  struct Packet {
```

Comment:
Maybe not so related to this PR, but I would provide in-struct initialization here, that is
```C++
unsigned long id = 0;
...
unsigned char length = 0;
```